resources/js/components/charts/GroupedBarChart.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<h2>Vurderingen av modulen med tanke på: </h2>
+	<h3>Vurderingen av modulen med tanke på: </h3>
 	<div id="gbc"></div>
 	<section>
 		<subsection class="category"><b>Læring</b></subsection>
```

Comment:
One thing to note is that the subsection tag used in the template is not a valid HTML tag. It's possible that this is a custom component, but if it's not, it would be better to use a different HTML tag or a custom component with a more appropriate name.